src/client.cpp

```diff
@@ -527,8 +527,8 @@ ProcessResponseEnum HttpClient::ProcessResponse(Value& result, bool notification
 bool TcpClient::GenerateHeader()
 {
     log_trace();
-    header_ << request_.Length() << ":";
-    log_debug("Request length=" << request_.Length());
+    header_ << static_cast<uint64_t>(request_.Length()) << ":";
```

Comment:
On OS X you have this:

``` typedef __darwin_size_t        size_t;  ```

Accordingly the size_t seems to be a different type (instead of typedef) on OS X.